src/dynatrace/opentelemetry/metrics/export/__init__.py

```diff
@@ -61,6 +44,44 @@ def configure_dynatrace_exporter(
         export_dynatrace_metadata: Optional[bool] = False,
         export_interval_millis: Optional[float] = None
 ):
+    """
+    Configures and creates a PeriodicExportingMetricReader and
+    DynatraceMetricsExporter combination.
+
+    Parameters
+    ----------
+    endpoint_url: str, Optional
+        The endpoint to send metrics to. Given an environment ID `myenv123` on
+        Dynatrace SaaS, the endpoint_url would be
+        `https://myenv123.live.dynatrace.com/api/v2/metrics/ingest`.
+        (default: local OneAgent Endpoint)
+    api_token: str, Optional
+        The API token for your Dynatrace environment with at least the scope
+        `metrics.ingest`.
+        (default: no API token).
+    prefix: str, Optional
+        Will be prepended to each metric key, separated by a dot
+        (`<prefix>.<namespace>.<name>`).
```

Comment:
```suggestion
        (`<prefix>.<name>`).
```
Otherwise people might be wondering where the `<namespace>.` is coming from

Reply:
thx. removed in [2f5bcd7](https://github.com/dynatrace-oss/opentelemetry-metric-python/pull/27/commits/2f5bcd7c53a60ca3de780552d59598a7071c66ee). also removed it from the readme where I copied it from.